apps/src/templates/sectionAssessments/MultipleChoiceByStudentSection.jsx

```diff
@@ -0,0 +1,41 @@
+import React, {Component, PropTypes} from 'react';
+import StudentAssessmentOverviewTable from './StudentAssessmentOverviewTable';
+import { studentAnswerDataPropType, questionStructurePropType } from './assessmentDataShapes';
+import {
+  getMultipleChoiceStructureForCurrentAssessment,
+  getStudentMCResponsesForCurrentAssessment,
+} from './sectionAssessmentsRedux';
+import { connect } from 'react-redux';
+
+class MultipleChoiceByStudentSection extends Component {
+  static propTypes = {
+    multipleChoiceStructure: PropTypes.arrayOf(questionStructurePropType),
+    studentAnswerData: PropTypes.arrayOf(studentAnswerDataPropType),
```

Comment:
Naming nit: Words like "structure" and "data" don't tell me a whole lot about the contents of these variables, and I'd omit them when possible.  I wonder if something like `questions` and `studentAnswers` is sufficient in this context?